### ISSUE TITLE
[Magier] Magische Vereinnahmung falsches Verhalten

### ISSUE THREAD

**Issue by Saguno** [ClassicWoW/Nefarian_1.12.1_Bugtracker#1952]
Welches Verhalten wird beobachtet?
Teilweise wird der Effekt durch absorbierten Schaden ausgelöst (siehe Screenshot), teilweise richtigerweise durch widerstandenen Schaden - auch nicht jedesmal, auch außerhalb des 1s cooldowns.
Wie sollte es sich verhalten?
Effekt sollte laut Beschreibung nur bei komplett widerstandenen Zaubern ausgelöst werden.
Schritte zur Reproduzierung?
Bei geskilltem Talent, Schadenszauber mittels Absorptionseffekten abwehren.
Zusätzliche Informationen (Screenshots, Videos, etc.)
http://www.directupload.net/file/d/4605/dmmlxiqu_jpg.htm
Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.
http://db.vanillagaming.org/?spell=29447
http://db.vanillagaming.org/?spell=20629

**Comment by dagochen** [comment]
Die Links gehen nicht und sind zur falschen DB :)

**Comment by Healchen** [comment]
bei mir gehen die links

**Comment by dagochen** [comment]
http://datenbank.classic-wow.org/?spell=29442

**Comment by Kriptyk** [comment]
Was man an dieser Stelle noch erwähnen sollte: Es gibt viele Spells, die gar keinen Proc auslösen trotz eines vollständigen Resists. Eine Liste welche Spells das sind hab ich nicht, da es auch recht viele sind und ich noch kein Muster entdecken konnte welche Spells betroffen sind und welche nicht und ob es überhaupt ein Muster gibt oder ob der Bug zufällig auftritt.

**Comment by Saguno** [comment]
Stimmt, das ist mir in letzter Zeit auch aufgefallen. Glaube das liegt aber nicht an speziellen Spells sondern muss einen anderen technischen Hintergrund haben.
